config/environments/development.rb

```diff
@@ -26,11 +26,21 @@
     config.cache_store = :null_store
   end
 
-  # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.raise_delivery_errors = true
   config.action_mailer.perform_caching = false
   config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
-  
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+      domain: ENV['MAILGUN_DOMAIN'],
+      address: 'smtp.mailgun.org',
+      port: 587,
+      user_name: "postmaster@#{ENV['MAILGUN_DOMAIN']}",
+      password: ENV['MAILGUN_PASSWORD'],
+      authentication: 'plain',
+      enable_starttls_auto: true
+  }
```

Comment:
It'll be clearer what is generic once I figure out what this will look like in prod. In test, you don't want this, instead you want it to use the :test delivery_method.